applications/application-life-cycle-management/odc-viewing-application-composition-using-topology-view.adoc

```diff
@@ -11,6 +11,7 @@ To view your applications in the *Topology* view and interact with them, ensure
 
 * You have xref:../../web-console/web-console.adoc#web-console[logged in to the web console].
 * You are in the xref:../../web-console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have the appropriate link:https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#default-roles_using-rbac[roles and permissions] in a project to create applications and other workloads in {product-title}.
```

Comment:
Same for this link. Update to this format:

xref:../authentication/using-rbac.adoc#default-roles_using-rbac[roles and permissions]